rancher2-deployment/Dockerfile

```diff
@@ -1,23 +1,57 @@
 FROM rancher/cli2 AS release
-ARG DEPLOYMENT_TEMPLATE_FILE="deployment.tmpl.yml"
-ENV DEPLOYMENT_TEMPLATE_FILE=${DEPLOYMENT_TEMPLATE_FILE}
 
-# Mandatory rancher context:
+# This image's main purpose is to run deployments to a Rancher2 cluster. It
+# leverages the rancher2-cli, kubectl and gomplate to accomplish that.
```

Comment:
what is gomplate?

Reply:
A flexible commandline tool for template rendering.  https://github.com/hairyhenderson/gomplate

Reply:
This is explained in README.md. Thus I don't see a reason to further describe it here.